docker-compose.yml

```diff
@@ -1,7 +1,9 @@
 services:
   hydra:
     container_name: hydra
-    image: ghcr.io/hydrasco/hydra:latest
+    build:
+      dockerfile: ./Dockerfile.latest
```

Comment:
We wouldn't need a Dockerfile to build it again with docker-compose because we are using `docker bake` to build in https://github.com/HydrasCo/hydra/blob/main/docker-bake.hcl already. If we want to pull the latest image from ghcr.io. We could `docker pull ghcr.io/hydrasco/hydra:latest` and `docker compose up` would use the latest pulled-down image

Reply:
🤷‍♂️ that does seem to work... @ljcesca any thoughts? I was mostly a keyboardist on this PR.

Reply:
🤷‍♂️ I'm indifferent. I could see us wanting more knobs for local testing eventually, but either would work for now.